Tests/Functional/lit.cfg

```diff
@@ -114,21 +114,14 @@ else:
                 '-I', libdispatch_overlay_dir,
                 '-I', os.path.join(libdispatch_overlay_dir, 'swift'),
                 '-L', libdispatch_build_dir,
-                '-L', os.path.join(libdispatch_build_dir, 'src'),
```

Comment:
There aren't any libraries in this build directory on Android.

Reply:
This will break the Windows build, please retain this.

Reply:
Alright, that's what I was worried about, removed this change.